### PR TITLE
[spacey2025] Add spacey2025 strategy

### PR DESCRIPTION
**Pull request by chuang39**
Implemented #332
https://github.com/snapshot-labs/snapshot-strategies/issues/332

Spacey2025 strategy
Implement the voting power by different category of erc721. The voting power rules are like below

SPAY x1
LAND x600
Ticket x1500
Tower x6

### CODE REVIEW

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi please revert changes from package-lock.json file

**Comment by chuang39** [comment]
> Hi please revert changes from package-lock.json file

@ChaituVR Thanks for looking into the diff! Already updated as per your feedback.

File: src/strategies/spacey2025/index.ts [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import { getAddress } from '@ethersproject/address';
+import { subgraphRequest } from '../../utils';
+
+export const author = '2fd';
```
Comment (ChaituVR) [inline src/strategies/spacey2025/index.ts:4]:
```suggestion
export const author = 'chuang39';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by chuang39** [comment]
@ChaituVR thanks for fixing it! I noticed it seemed fixed by your commit already. I think I don't need to do anything for it, am I right? Let me know if any code I need to update.

File: src/strategies/spacey2025/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- Tower x6
+
+
+Here is an example of parameters:
+
+{
+    "address": "0x230185C3B02b897B89cb1e62717AD7772b8319DA",
+    "symbol": "SPACEY NFT",
+    "decimals": 18,
+}
```
Comment (ChaituVR) [inline src/strategies/spacey2025/README.md:16]:
```suggestion
{
    "address": "0x230185C3B02b897B89cb1e62717AD7772b8319DA",
    "symbol": "SPACEY NFT"
}
```

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**